tests/distributions/test_continuous.py

```diff
@@ -332,6 +332,13 @@ def test_chisquared_logcdf(self):
             lambda value, nu: st.chi2.logcdf(value, df=nu),
         )
 
+    def test_chisquared_icdf(self):
```

Comment:
No need to test, the Gamma is enough

Reply:
Pardon my lack of attention, I forgot to add this as well in previous commits. Fixed in https://github.com/pymc-devs/pymc/pull/6845/commits/e045748f94a51bf1d7905ff7526bbe82f03577e9